topics/dev/metadata.yaml

```diff
@@ -11,3 +11,23 @@ maintainers:
   - hexylena
 
 gitter: galaxyproject/dev
+
+subtopics:
```

Comment:
Really good idea.
However, I shouldn't have ordered like that but more like:
core -> API -> dependencies -> tooldev -> viz -> advanced